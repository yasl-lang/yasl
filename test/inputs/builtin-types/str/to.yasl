echo 'tobool:'
echo 'false'->tobool()
echo 'fassd'->tobool()
echo ''->tobool()
echo 'tobyte:'
echo .A->tobyte()
echo .a->tobyte()
echo ' '->tobyte()
echo '~'->tobyte()
echo 'tolist:'
echo 'abcdefg'->tolist()
echo 'abcdefg'->tolist(2)
echo ''->tolist()
echo 'tolower:'
echo 'YASL'->tolower()
echo 'Yasl'->tolower()
echo '!@#$'->tolower()
echo 'toupper:'
echo 'yasl'->toupper()
echo 'Yasl'->toupper()
echo '%^&*'->toupper()
