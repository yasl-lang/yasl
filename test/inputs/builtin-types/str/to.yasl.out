tobool:
true
true
false
tobyte:
65
97
32
126
tolist:
[a, b, c, d, e, f, g]
[ab, cd, ef, g]
[]
tolower:
yasl
yasl
!@#$
toupper:
YASL
YASL
%^&*
